### PR TITLE
chore: add missing french translations

### PR DESCRIPTION
**Pull request by Joluma**
## Description
- add translations for French.
- use main `fr` instead of `fr-CA`
- remove redundant translation key and reorder file to make comparaison easier with original language

## Impacted Areas in the application
The french translations

## Testing
Open the application and switch language to french.

### CODE REVIEW

File: src/locales/fr/translation.json [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
+  "Country": "Pays",
   "Country to determine the age distribution in the population": "Pays pour déterminer la répartition par âge dans la population",
   "Critical": "Critique",
-  "CSV file could not be parsed": "Fichier CSV ne peut pas être lu",
-  "CSV file is missing": "Fichier CSV manquant",
   "Cumulative confirmed cases": "Cas confirmés cumulés",
   "Cumulative confirmed deaths": "Morts confirmées cumulées",
   "Cumulative deaths": "Morts cumulées",
-  "Drag n' drop some files here, or click to select files": "Déposez des fichiers ici, ou cliquez pour selectionner des fichiers",
+  "Custom": "Customisé",
```
Comment (whiver) [inline src/locales/fr/translation.json:33]: Anglicism: *Personnalisé* would be better IMO

File: src/locales/fr/translation.json [hunk 2]
```diff
@@ -1,37 +1,48 @@
...
+  "Default": "Défaut",
+  "Distribution across age groups": "Distribution à travers les groupes d'âge",
```
Comment (whiver) [inline src/locales/fr/translation.json:35]: *Tranches d'âge* ?

File: src/locales/fr/translation.json [hunk 3]
```diff
@@ -1,37 +1,48 @@
...
   "Drag black dots with the mouse to simulate how infection control affects the outbreak trajectory. One is no infection control, zero is complete prevention of all transmission.": "Faites glisser les points noirs avec la souris pour simuler comment le contrôle des infections affecte la trajectoire de l'épidémie. 1 correspond à aucun contrôle des infections, 0 à une prévention complète de toute transmission.",
+  "Drag n' drop some files here, or click to select files": "Déposez des fichiers ici, ou cliquez pour selectionner des fichiers",
   "Drop the files here": "Déposez les fichiers ici...",
   "Epidemiological parameters specifing growth rate, seasonal variation, and duration of hospital stay. The presets are combinations of speed and geography (speed/region).": "Paramètres épidémiologiques spécifiant le taux de croissance, la variation saisonnière et la durée du séjour à l'hôpital. Les préréglages sont des combinaisons de vitesse et de géographie (vitesse / région).",
   "Epidemiology": "Épidémiologie",
   "Error": "Erreur",
   "Export": "Export",
+  "Fast/North": "Rapide/Nord",
+  "Fast/South": "apide/Sud",
```
Comment (whiver) [inline src/locales/fr/translation.json:44]: typo

File: src/locales/fr/translation.json [hunk 4]
```diff
@@ -41,26 +52,32 @@
...
+  "Isolated": "Isolé",
   "Latency": "Latence",
   "Loading...": "Chargement...",
   "Log scale": "Échelle logarithmique",
-  "Mild": "Doux",
+  "Mild": "Léger",
   "Mitigation": "Atténuation",
   "Modal title": "Titre modal",
+  "Moderate mitigation": "Atténuation modérée",
+  "Moderate/North": "Modéréé/Nord",
```
Comment (whiver) [inline src/locales/fr/translation.json:63]: typo (same line below)

File: src/locales/fr/translation.json [hunk 5]
```diff
@@ -76,32 +93,42 @@
...
   "Scenario": "Scenario",
   "Seasonal forcing": "Forçage saisonnier",
   "Seasonal peak": "Pic saisonnier",
   "Select region for which to plot confirmed case and death counts.": "Sélectionnez la région pour laquelle tracer les cas confirmés de cas et de décès.",
   "Severe": "Sévère",
-  "Severity-assumptions": "Hypothèses de gravité",
+  "Severely ill": "Gravement malade",
+  "Severity assumptions and age-specific isolation": "Hypothèses de gravité et isolation par âge",
   "Severity of ICU overflow": "Gravité du débordement des soins intensifs",
+  "Show humanized results": "Montrer les résultats humanisés",
```
Comment (whiver) [inline src/locales/fr/translation.json:105]: This is difficult to translate it directly: here "humanisés" doesn't make much sense. I would suggest simply *Mettre en forme les résultats*

File: src/locales/fr/translation.json [hunk 6]
```diff
@@ -41,26 +52,32 @@
...
   "Number of controllable points on the mitigation curve": "Nombre de points contrôlables sur la courbe d'atténuation",
   "Number of hospital beds available in health care system. Presets are rough estimates indicating total capacity. Number of beds available for COVID-19 treatment is likely much lower.": "Nombre de lits d'hôpital disponibles dans le système de soins de santé. Les préréglages sont des estimations approximatives indiquant la capacité totale. Le nombre de lits disponibles pour le traitement du COVID-19 est probablement beaucoup plus faible.",
   "Number of ICU/ICMUs available in health care system. Presets are rough estimates indicating total capacity. Number of ICU/ICMUs available for COVID-19 treatment is likely much lower.": "Nombre d'unités de soins intensifs disponibles dans le système de soins de santé. Les préréglages sont des estimations approximatives indiquant la capacité totale. Le nombre d'unités de soins intensifs / ICMU disponibles pour le traitement au COVID-19 est probablement beaucoup plus faible.",
+  "Number of people served by health care system.": "Nombre de personnes desservies par le système de santé.",
```
Comment (whiver) [inline src/locales/fr/translation.json:74]: *desservies* seems a bit strange to me, as the health care system isn't harming people :p What do you think of *couvertes* instead?

**Comment by rneher** [comment]
Thanks @Joluma @whiver. The "Fast/North" and "moderate Mitigation" are no longer needed as we moved to the new scenario dropdown in master. But if you resolve the other discussions, i'll go ahead and merge.

File: src/locales/fr/translation.json (hunk 6 above)
Comment (Joluma) [reply src/locales/fr/translation.json]: I change for `_prises en charge_`, I think it's more suitable.

**Comment by Joluma** [comment]
@whiver since there's not that much translations, I just checked manually, but as it grows I'll maybe try to write a simple script to check that.

**Comment by whiver** [comment]
Okay, thanks for the confirmation :)

File: src/locales/fr/translation.json [hunk 7]
```diff
@@ -41,26 +52,32 @@
...
   "Loading...": "Chargement...",
   "Log scale": "Échelle logarithmique",
-  "Mild": "Doux",
+  "Mild": "Léger",
   "Mitigation": "Atténuation",
   "Modal title": "Titre modal",
+  "Moderate mitigation": "Atténuation modérée",
+  "Moderate/North": "Modéré/Nord",
+  "Moderate/South": "Modéré/Sud",
+  "Moderate/Tropical": "Modéréé/Tropiques",
```
Comment (whiver) [inline src/locales/fr/translation.json:65]: typo

File: src/locales/fr/translation.json [hunk 8]
```diff
@@ -76,32 +93,42 @@
...
-  "This output of a mathematical model depends on model assumptions and parameter choices. We have done our best (in limited time) to check the model implementation is correct. Please carefully consider the parameters you choose and interpret the output with caution.": "Cette sortie d'un modèle mathématique dépend des hypothèses du modèle et des choix de paramètres. Nous avons fait de notre mieux (dans un temps limité) pour vérifier que la mise en œuvre du modèle est correcte. Veuillez considérer attentivement les paramètres que vous choisissez et interpréter la sortie avec prudence",
   "Run": "Lancer",
   "Scenario": "Scenario",
   "Seasonal forcing": "Forçage saisonnier",
   "Seasonal peak": "Pic saisonnier",
   "Select region for which to plot confirmed case and death counts.": "Sélectionnez la région pour laquelle tracer les cas confirmés de cas et de décès.",
   "Severe": "Sévère",
-  "Severity-assumptions": "Hypothèses de gravité",
+  "Severely ill": "Gravement malade",
+  "Severity assumptions and age-specific isolation": "Hypothèses de gravité et isolation par âge",
```
Comment (whiver) [inline src/locales/fr/translation.json:103]: "isolation par âge" doesn't sound natural to me. Looking at the corresponding section, this seems to show the severity for each age group, so I would translate it with something like *Hypothèses de gravité par tranche d'âge*. What do you think?

File: src/locales/fr/translation.json [hunk 9]
```diff
@@ -113,10 +140,13 @@
...
+  "This table summarizes the assumptions on severity which are informed by epidemiological and clinical observations in China. The first column reflects our assumption on what fraction of infections are reflected in the statistics from China, the following columns contain the assumption on what fraction of the previous category deteriorates to the next. These fields are editable and can be adjusted to different assumptions. The last column is the implied infection fatality for different age groups.": "Ce tableau résume les hypothèses de gravité qui sont basées sur des observations épidémiologiques et cliniques en Chine. La première colonne reflète notre hypothèse sur la fraction d'infections reflétée dans les statistiques de la Chine, les colonnes suivantes contiennent l'hypothèse sur la fraction de la catégorie précédente qui se détériore dans la suivante. Ces champs sont modifiables et peuvent être ajustés selon différentes hypothèses. La dernière colonne est la mortalité implicite d'infection pour différents tranches d'âge.",
```
Comment (whiver) [inline src/locales/fr/translation.json:148]: typo: *pour différents tranches d'âge* -> *pour les différentes tranches d'âge*

**Comment by ivan-aksamentov** [comment]
@Joluma @whiver Merci! Let me know when this is ready.

@whiver There are several tools to extract the translations from i18next:
https://react.i18next.com/guides/extracting-translations

We also plan to integrate with Locize, but keep forgetting to open the account :)

Any help is appreciated!

**Comment by whiver** [comment]
> 
> 
> @Joluma @whiver Merci! Let me know when this is ready.
> 
> @whiver There are several tools to extract the translations from i18next:
> https://react.i18next.com/guides/extracting-translations
> 
> We also plan to integrate with Locize, but keep forgetting to open the account :)
> 
> Any help is appreciated!

I saw these tools (I actually mentioned them in #56) but it doesn't seem like they are currently integrated in the codebase, right?
Indeed being integrated with a localization platform would be awesome! It doesn't seem like Locize is free for Open source projects, does it?

**Comment by ivan-aksamentov** [comment]
@whiver 
> Locize is free for Open source projects, does it?

Does not seem like it, but I spoke with their person and they may give us a waiver, especially if we host everything ourselves. Which I want to do anyways.

Long story short, we've created an account but are super busy with the features for the next release, so if you have resources to investigate proper i18n workflow for devs and translators  that't be great. 

The goal is to automate as much as possible, make it easy for non-tech person to amend translations and to use as little as possible (or none) external network requests. So, cli, nice UI, and Github bots probably.

**Comment by whiver** [comment]
> 
> 
> @whiver
> 
> > Locize is free for Open source projects, does it?
> 
> Does not seem like it, but I spoke with their person and they may give us a waiver, especially if we host everything ourselves. Which I want to do anyways.
> 
> Long story short, we've created an account but are super busy with the features for the next release, so if you have resources to investigate proper i18n workflow for devs and translators that't be great.
> 
> The goal is to automate as much as possible, make it easy for a non-tech person to amend translations and to make as little as possible (or none) external network requests in production. So, cli, nice UI, and Github bots probably.

Sounds good! I may look at that after I finish working on the import feature.

File: src/locales/fr/translation.json [hunk 10]
```diff
@@ -113,10 +140,13 @@
...
+  "This table summarizes the assumptions on severity which are informed by epidemiological and clinical observations in China. The first column reflects our assumption on what fraction of infections are reflected in the statistics from China, the following columns contain the assumption on what fraction of the previous category deteriorates to the next. These fields are editable and can be adjusted to different assumptions. The last column is the implied infection fatality for different age groups.": "Ce tableau résume les hypothèses de gravité qui sont basées sur des observations épidémiologiques et cliniques en Chine. La première colonne reflète notre hypothèse sur la fraction d'infections reflétée dans les statistiques de la Chine, les colonnes suivantes contiennent l'hypothèse sur la fraction de la catégorie précédente qui se détériore dans la suivante. Ces champs sont modifiables et peuvent être ajustés selon différentes hypothèses. La dernière colonne est la mortalité implicite d'infection pour différentes tranches d'âge.",
```
Comment (whiver) [inline src/locales/fr/translation.json:148]: *les colonnes suivantes contiennent l'hypothèse sur la fraction de la catégorie précédente qui se détériore dans la suivante*: that sounds odd. I would rephrase it more drastically. Maybe *les colonnes suivantes représentent quelle fraction des cas de la catégorie précédente se sont aggravés*?
